js/app.js

```diff
@@ -0,0 +1,96 @@
+
+
+$(document).ready(function(){
+
+	// Changes Sidebar text and changes button to start button
+	$(document).on('click', '.nextButton', function(){
+		$(".introText").remove();
+		$(".introduction").append("<p class='introText'> Answer four short questions, and we will serve ye up something to yer liken! </p>")
+		$(".nextButton").addClass("startButton").val("Start").removeClass("nextButton");
+	});
+
+	// Uses start button to initiate the game
+	$(document).on('click', '.startButton', function(){
+		$(".dim").removeClass("dim");
+	});
+
+
+
+// Empty array to fill with drink ingredients
+var drink = [];
+
+// Empty array 
+var drinkName = [];
+
+// Ingredients for the drinks
+var ingredients = {
+	strong: ['Glug of Rum', 'Slug of Whisky', 'Shot of Jager'],
+	weak: ['Splash of Rum','Ounce of Whisky','Shot of Gin'],
+	bitter: ['Shake of Bitters', 'Splash of Tonic', 'Twist of Lemon Peel'],
+	bland: ['Some Carbonated Soda','Tonic Water','Hint of RedBull'],
+	salty: ['Olive on a Stick', 'Salt-Dusted Rim', 'Rasher of Bacon'],
+	sweet: ['Sugar Cube', 'Spoonful of Honey', 'Splash of Cola'],
+	fruity: ['Splash of Apple Juice', 'Strawberry Syrup', 'Raspberry Syrup'],
+	vegetable: ['Hint of Tomato','Celery Stick','Hint of Carrot Juice'],
+	drink: function(type) {
+		drink.push(ingredients[type][(Math.floor(Math.random() * ingredients[type].length))]);
+	}
+}
+
+// Names for the drinks
+var names = {
+	first:['Fluffy', 'Hardy', 'Sweet', 'Gnarly'],
+	middle: ['Puff-', 'Sea-', 'Scruff-', 'Pirate-'],
+	last: ['Dog','Badger','Horse','Fish'],
+	full: function(type) {
```

Comment:
When you call this, you have to do it 3 times to make the name. Consider ma king the function handle building all 3 pieces with a single call